public_html/wp-content/plugins/camptix/addons/require-login.php

```diff
@@ -121,7 +121,7 @@ public function ticket_form_message() {
 		}
 
 		// Warn users that they will need to login to purchase a ticket
-		if ( ! is_user_logged_in() ) {
+		if ( $camptix->has_tickets_available() && ! is_user_logged_in() ) {
```

Comment:
we should probably check the status of the WordCamp as well, as an event can end with tickets still available.

Reply:
oh, I guess it could rely on https://github.com/WordPress/wordcamp.org/pull/1404/

Reply:
Good call.

Reply:
Actually, we don't need to check the WordCamp status. When a camp is closed, tickets are all set to unavailable.